manager/manifests/operator.yaml

```diff
@@ -41,21 +41,30 @@ metadata:
   namespace: $CORTEX_NAMESPACE
   labels:
     workloadType: operator
+    workloadID: operator
+    app: operator
```

Comment:
Is the `app` label necessary? Does istio use it? If not can we delete it?

Reply:
istio uses it